src/Controller/ActivityTypeController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\ActivityType;
+use App\Repository\TypeRepository;
+use FOS\RestBundle\Controller\Annotations\Route;
+use JMS\Serializer\SerializationContext;
+use JMS\Serializer\SerializerInterface;
+use Nelmio\ApiDocBundle\Annotation\Model;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use FOS\RestBundle\Controller\Annotations as Rest;
+use Swagger\Annotations as SWG;
+
+/**
+ * ActivityType controller.
+ * @Route("/api/types", name="activityTypes")
+ */
+class ActivityTypeController extends AbstractController
+{
+    /**
+     * @var TypeRepository
+     */
+    private $typeRepository;
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    public function __construct(TypeRepository $typeRepository, SerializerInterface $serializer)
```

Comment:
Shouldn't `TypeRepository` be `ActivityTypeRepository`? I thought it was renamed.